platform/ext/target/nordic_nrf/nrf54l15dk_nrf54l15_cpuapp/config.cmake

```diff
@@ -0,0 +1,12 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2024, Nordic Semiconductor ASA.
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+#-------------------------------------------------------------------------------
+
+include(${PLATFORM_PATH}/common/nrf54l15/config.cmake)
+
+set(SECURE_UART1                        OFF         CACHE BOOL      "Enable secure UART1" FORCE)
+set(SECURE_UART30                       OFF         CACHE BOOL      "Enable secure UART1" FORCE)
+set(BL2                                 OFF         CACHE BOOL      "Enable secure UART1" FORCE)
```

Comment:
comment fails + no newline at end of file (and in `cpuarch.cmake`)

Reply:
You are right that this is of course wrong,  and the same goes for your comment in the tfm_platform_system.c. I think that the upstream PR is about to get merged now. If it does I will just create another PR to fix these things upstream directly.